### ISSUE TITLE
Possible suggestion: SQRL

### ISSUE THREAD

**Issue by mbennett417** [akeeba/loginguard#52]
Just an inquiry, wondering if you've investigated integrating this with SQRL?
https://www.grc.com/sqrl/sqrl.htm

I see on the links at the bottom of the page that both Drupal and Wordpress have working implementations, and at first and second glance it seems that some of the goals of Loginguard parallel SQRL.

**Comment by nikosdion** [comment]
No. It’s a replacement for authentication (replaces username and password). It’s not a second factor authentication method. 

Conceptually it’s like logging in with Facebook, Twitter or Google. That is, instead of entering your username and password on each individual site you do it in one central location and then use a web based method to provide an authentication token from the central location to each site you want to log in. To that end it’d be a fit for SocialLogin which provides alternate first authentication methods. Unfortunately, we have already discontinued that product due to lack of available time and a use case for our own web properties.

**Comment by mbennett417** [comment]
That makes sense.  Sometimes it's hard reading a description of a product or idea to figure out what it really does, or what it's similar to.  Thanks for the reply.
